app/assets/javascripts/activeadmin_addons/select2.js

```diff
@@ -163,6 +153,18 @@ $(function() {
           });
         }
 
+        var select2Config = {
```

Comment:
All 'var' declarations must be at the top of the function scope  vars-on-top<br>Unexpected var, use let or const instead                         no-var